packages/shared/src/components/MainLayout.tsx

```diff
@@ -52,11 +55,18 @@ export default function MainLayout({
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { user, showLogin, loadingUser } = useContext(AuthContext);
   const [showGreeting, setShowGreeting] = useState(false);
+  const showSidebar = useMedia(
```

Comment:
Maybe make this sidebarVisible? because that's actually what's happening
Or sidebarRendered